core/src/main/java/org/libreoffice/lots/db/Search.java

```diff
@@ -180,9 +178,9 @@ private static String formatQuery(String query)
   }
 
   /**
-   * Nimmt ein Template für eine Suchanfrage entgegen (das Variablen der Form "${suchanfrageX}"
-   * enthalten kann) und instanziiert es mit Wörtern aus words, wobei nur die ersten wordcount
-   * Einträge von words beachtet werden.
+   * Takes a template for a search query (which may contain variables of the form "${suchanfrageX}")
+   * and instantiates it with words from words, considering only the first wordcount
+   * entries of words.
```

Comment:
"suchanfrageX" should also be translated, such as "searchqueryX".

> instantiates it with words from words

That's difficult to parse in English - in German you knew that the English term refers to the variable name, but in pure English that context gets lost. How about "instantiates it with a list of words"?